index.html

```diff
@@ -168,17 +168,18 @@ <h1 class="orange crete-f f-weight400 l-height-88 f-size-72 ">About Me</h1>
 
         <form action="https://formspree.io/f/meqdkpzn" method="POST">
             <label>
-                <input type="name" name="name" placeholder="Full name" maxlength="30">
+                <input type="name" ID="name" name="name" placeholder="Full name" maxlength="30">
             </label>
             <hr class="rule-5">
             <label>
-                <input type="email" name="email" placeholder="Email address">
+                <input type="email" id="email" name="mail" placeholder="Email address">
             </label>
             <hr class="rule-5">
             <label>
-                <textarea name="message" placeholder="Enter text here" maxlength="500"></textarea>
+                <textarea name="message" id="message-text" placeholder="Enter text here" maxlength="500"></textarea>
             </label>
             <button type="submit" class="white-1 inter-f f-weight700 l-height-24 f-size-17">Get in touch</button>
+            <small class="msg"></small>
```

Comment:
[OPTIONAL] You can consider styling the error message with red or something similar to make it more visible and up to standard since most error messages are red/danger in color. :red_circle: